src/core/LanguageMenu/LanguageMenu.baseStyles.tsx

```diff
@@ -13,19 +13,24 @@ export const baseStyles = withSuomifiTheme(
     &.fi-language-menu-language_button {
       ${element({ theme })}
       ${theme.typography.actionElementInnerTextBold}
-      padding: 5px ${theme.spacing.insetXs} 5px ${theme.spacing.insetM};
-      line-height: 28px;
+      padding: ${theme.spacing.xs};
```

Comment:
Not quite sure if we did. With the latest push the values that can use tokens are using tokens. Many of the values are are not found in inset tokens unfortunately. Should we use pixel values instead of regular (non-inset) tokens here?

Reply:
My personal opinion is that we should prefer tokens when ever available, and then em values and lastly pixel values. In our case there is not much difference between em/px since we have quite a lot of pixel values in e.g. typography tokens. Let's do an inventory of used margins, borders and paddings after some time to see if we can add order to the current system. Until that, I'd propose to use what works the best for the case.